frontend/pages/[path].tsx

```diff
@@ -0,0 +1,53 @@
+import { StaticPage } from "@/interfaces";
+import { FC } from "react";
+
+import Layout from "@/components/Layout";
+import { Container, useMediaQuery } from "@material-ui/core";
+import axios from "axios";
+
+interface StaticPageProps {
+  page: StaticPage;
+}
+
+const StaticPage: FC<StaticPageProps> = ({ page }: StaticPageProps) => {
+  // check if the viewport width is less than 600px
+  const isSmall = useMediaQuery("@media (max-width:600px)");
+
+  return (
+    <Layout>
```

Comment:
@mattfredericksen, looks like this can't be compiled without the `title` attribute on Layout

Reply:
Fixed in 4cbc0a7.